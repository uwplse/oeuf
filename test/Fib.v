Require List SourceLang Pretty OeufPlugin.OeufPlugin.
Import List.ListNotations.

Eval compute Then Write To File "fib.oeuf" (Pretty.expr.print (@SourceLang.fib_reflect [])).
