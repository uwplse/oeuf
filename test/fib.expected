fib(0) = 0
fib(1) = 1
fib(2) = 1
fib(3) = 2
fib(4) = 3
fib(5) = 5
fib(6) = 8
fib(7) = 13
